pkg/reconciler/delivery/delivery.go

```diff
@@ -47,6 +47,8 @@ const (
 	KCDNamespace = "knative-serving"
 	// KCDName is the name of this project
 	KCDName = "knative-continuous-delivery"
+	// PolicyNameKey is the key that maps to the name of a policy in the annotations of a Service/Configuration
+	PolicyNameKey = "RolloutPolicyName"
```

Comment:
This should go in register.go and have a structure similar to knative serving. https://github.com/knative/serving/blob/8407cc746fb4453cd2d2b5abfa687843867600dd/pkg/apis/serving/register.go#L30

Reply:
fixed